llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -135,6 +135,43 @@ def _register_model_with_run_id_multiprocess(
     )
 
 
+def _log_model_multiprocess(
+    mlflow_logger: MLFlowLogger,
+    composer_logging_level: int,
+    input_example: dict[str, Any],
+    task: str,
+    name: str,
+    model_name: str,
+    await_creation_for: int,
+):
+    """
+    Call MLFlowLogger.log_model.
+    
+    Used mainly to log from a child process.
+    """
+    # Setup logging for child process. This ensures that any logs from composer are surfaced.
+    if composer_logging_level > 0:
+        # If logging_level is 0, then the composer logger was unset.
+        logging.basicConfig(
+            format=
+            f'%(asctime)s: rank{dist.get_global_rank()}[%(process)d][%(threadName)s]: %(levelname)s: %(name)s: %(message)s',
+        )
+        logging.getLogger('composer').setLevel(composer_logging_level)
+    mlflow_logger.log_model(
+        flavor='transformers',
+        artifact_path="model",
+        input_example=input_example,
+        task=task,
+        metadata={
+            "task": task,
+            "databricks_model_source": "genai-fine-tuning",
```

Comment:
the metadata should not be hardcoded, it should be passed in like with the existing register call